### PR TITLE
Feature add product

### PR DESCRIPTION
**Pull request by at-ngatran**
I have already create new feature add product
Please review and merge this branch.
Thanks and Best Regards

### CODE REVIEW

File: app/Http/Controllers/ProductController.php [hunk 1]
```diff
@@ -2,23 +2,60 @@
 
 namespace App\Http\Controllers;
 
-use Illuminate\Http\Request;
-
+//use Illuminate\Http\Requestsiuse App\Repositories\ProductRepository as Product;
```
Comment (at-anhnt2) [inline app/Http/Controllers/ProductController.php]: [rank C] why don't you remove this line?

File: resources/views/product/create.blade.php [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+            </div>
+          </div>
+           <div class="item form-group">
+            <label class="control-label col-md-3 col-sm-3 col-xs-12" for="is_on_sale">@lang('products.label_is_on_sale_product')
+              <span class="required">@lang('products.required_product')</span>
+            </label>
+            <div class="col-md-6 col-sm-6 col-xs-12">
+              <select autofocus name="is_on_sale">
+                <option value="1">@lang('products.option_sale_Yes_product')</option>
+                <option value="0">@lang('products.option_sale_No_product')</option>
```
Comment (at-anhnt2) [inline resources/views/product/create.blade.php]: [rank B] Please avoid hardcode, the option values `0` and `1` should be defined as constants.

File: app/Http/Controllers/ProductController.php [hunk 3]
```diff
@@ -2,23 +2,60 @@
...
+     *
+     * @param Request $request request
+     *
+     * @return \Illuminate\Http\Response
+    */
+    public function store(ProductRequest $request)
     {
-        $products = Product::all();
+        try {
+             $product = new Product;
```
Comment (dzungnguyenh) [inline app/Http/Controllers/ProductController.php]: [rank C] please use create form of Laravel, we don't need to assign one by one for it.

**Comment by at-ngatran** [comment]
I have fixed your comments, please help me review and merge .
thank you

File: resources/views/product/create.blade.php [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                    <option value="{{ $category->id }}">{{ $category->name }}</option>
+                  @endforeach
+              </select>
+            </div>
+          </div>
+          <div class="item form-group">
+            <label class="control-label col-md-3 col-sm-3 col-xs-12" for="number"> @lang('products.label_price_product') <span class="required">@lang('products.required_product')</span>
+            </label>
+            <div class="col-md-6 col-sm-6 col-xs-12">
+              <input type="number" id="number" name="price" required="required" data-validate-min="1,200000" class="form-control col-md-7 col-xs-12" placeholder="000.000 vnd">
```
Comment (dzungnguyenh) [inline resources/views/product/create.blade.php]: [rank B] you have to define min, max values as common constant.
All placeholder need define in language file too.

File: resources/views/product/create.blade.php [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+            </label>
+            <div class="col-md-6 col-sm-6 col-xs-12">
+              <input type="number" id="number" name="remaining_amount" required="required" data-validate-minmax="1,800" class="form-control col-md-7 col-xs-12" placeholder="@lang('products.label_remaining_amount_product')">
+            </div>
+          </div>
+          <div class="item form-group">
+            <label for="descrition" class="control-label col-md-3 col-sm-3 col-xs-12">@lang('products.label_description_product')
+            </label>
+            <div class="col-md-6 col-sm-6 col-xs-12">
+              <textarea id="description" type="text" name="description" data-validate-length-range="6,32" class="form-control col-md-7 col-xs-12" placeholder="Description"></textarea>
```
Comment (dzungnguyenh) [inline resources/views/product/create.blade.php]: [rank B]All placeholder need define in language file too.

**Comment by at-ngatran** [comment]
I have fixed your comments, please help me review and merge .
thanks so much

**Comment by dzungnguyenh** [comment]
So good!

**Pull request merged**